engine/script/src/http_service.cpp

```diff
@@ -269,7 +269,7 @@ namespace dmHttpService
             if (message->m_Descriptor == (uintptr_t) dmHttpDDF::HttpRequest::m_DDFDescriptor)
             {
                 dmHttpDDF::HttpRequest* request = (dmHttpDDF::HttpRequest*) &message->m_Data[0];
-                HandleRequest(worker, &message->m_Sender, request);
+                HandleRequest(worker, &message->m_Sender, 0, message->m_UserData2, request);
```

Comment:
If it's set, we pass it to the send response, which will post the user data back to the receiver.